qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ReportHistoryPage.java

```diff
@@ -21,6 +22,7 @@ public ReportHistoryPage(Page page) {
 	}
 
 	public RenderDefaultReportPage clickOnViewLink() {
+		waiter.until(ExpectedConditions.visibilityOfElementLocated(VIEW_REPORT));
```

Comment:
Temporarily comment out the above line and i see what happens?

Reply:
Finished @dkayiwa

Reply:
@sherrif10 does it still fail?

Reply:
Nope it doesnt . Becuase i changed local.test.properties to run on qa server. but when i change it to local server it will fail.

Reply:
Change it to the local server for me to confirm.